src/shared/config.ts

```diff
@@ -167,6 +180,30 @@ export const ConvictConfig = Convict<ServiceConfig>({
       format: 'port',
       default: 3002,
       env: 'OUTBOUND_PORT'
+    },
+    TLS: {
+      mutualTLS: {
+        enabled: false
+      },
+      creds: {
+        ca: '',
+        cert: '',
+        key: ''
+      }
+    }
+  },
+  CONTROL: {
+    MGMT_API_WS_URL: {
```

Comment:
Do we need any TLS config for MGM API?

Reply:
Not that I know of based off of looking at `mojaloop-connector`